### PR TITLE
chore: Landing page content updates and improvements for content rendering flow

### PR DESCRIPTION
**Pull request by ala-n**
In bounds of this PR:
 - Add support for `.yml` data content declaration for 11ty renderer
 - Rendering flow improvements: 
    - make landing components content-less
    - create `index.11tydata.yml` as a content holder for landing page
 - Make footer year dynamic, based on the build time
 - Add additional links support for the landing footer section
 - Replace browser support section arrow-link with a button style
 - Fix Marquee banner content alignment, add optional `slogan` section
 - Content updates (see original [content proposal](https://docs.google.com/document/d/1WlyBAg7h5vqI5V_8ICpk-z6SZzQpZk6AKrYO5uGvSwY/edit?usp=sharing) )
 
 To preview current landing content see [index.11tydata.yml](https://github.com/exadel-inc/esl/tree/chore/landing-seo/pages/views/index.11tydata.yml)
 Or see the following screenshots to check it out (I hope I will support them up to date)
 
 
image
image
image
image
image

Closes: #636

### CODE REVIEW

**Review by NastaLeo — APPROVED** [review]

**Review by vbaranov-exadel — APPROVED** [review]
I have not enough skills to make a substantial review, but the description of the objective to this pull request seems to be reasonable to me.

File: pages/views/index.11tydata.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          cutting out the unnecessary code that takes a toll on user experience.
+      - icon: benefits/powerful.svg
+        name: Powerful
+        text: >
+          While the modules in our web components library may be small, they’re also universal,
+          allowing you to mix and match the UI components at will to create large modules.
+      - icon: benefits/tested.svg
+        name: Tested
+        text: >
+          We test all of our utilities with unit tests to make ESL the best front end component library,
```
Comment (fshovchko) [inline pages/views/index.11tydata.yml:48]:
```suggestion
          We cover all of our utilities with unit tests to make ESL the best front end component library,
```

**Review by fshovchko — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

File: pages/views/index.11tydata.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        style: sec-orange
+
+  benefits:
+    title: Why the ESL Web Components Library?
+    items:
+      - icon: benefits/efficient.svg
+        name: Efficient
+        text: >
+          Our team has reduced the work needed to create reusable UI components by giving you a reliable base
+          to customize and expand on for fast and accurate development.
```
Comment (fshovchko) [inline pages/views/index.11tydata.yml:29]:
```suggestion
          Our team goal is to reduce development time by providing heavily customizable and expandable UI components
```

Comment (fshovchko) [inline pages/views/index.11tydata.yml:29]:
```suggestion
          Our team goal is to reduce development time by providing heavily customizable and expandable UI components
```

Comment (fshovchko) [inline pages/views/index.11tydata.yml:29]:
```suggestion
          Our team goal is to reduce development time by providing heavily customizable and expandable UI components
```

Comment (fshovchko) [inline pages/views/index.11tydata.yml:29]:
```suggestion
          Our team goal is to reduce development time by providing heavily customizable and expandable UI components
```

File: pages/views/index.11tydata.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      - icon: benefits/fast.svg
+        name: Fast
+        text: >
+          With ESL, there’s no need to write lengthy markup — the reusable elements in our UI component library are small,
+          but they’re designed to be the optimal building blocks for your content.
```
Comment (fshovchko) [inline pages/views/index.11tydata.yml:34]:
```suggestion
          There’s no need to write lengthy markup — the reusable elements in our UI component library are small,
          but they’re designed to be the optimal building blocks for your content.
```

File: pages/views/index.11tydata.yml [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+      - icon: benefits/light.svg
+        name: Light
+        text: >
+          We specially designed our reusable web components to work quickly and efficiently in browsers,
+          cutting out the unnecessary code that takes a toll on user experience.
```
Comment (fshovchko) [inline pages/views/index.11tydata.yml:39]:
```suggestion
          We specially designed ESL's web components to work quickly and efficiently in browsers,
          cutting out the unnecessary code that takes a toll on user experience.
```

File: pages/views/index.11tydata.yml [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+      - icon: benefits/powerful.svg
+        name: Powerful
+        text: >
+          While the modules in our web components library may be small, they’re also universal,
+          allowing you to mix and match the UI components at will to create large modules.
```
Comment (fshovchko) [inline pages/views/index.11tydata.yml:44]:
```suggestion
          While our library is small, it's also universal, allowing you to mix and match the UI components at will to create large modules.
```

File: pages/views/index.11tydata.yml [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+      - icon: benefits/tested.svg
+        name: Tested
+        text: >
+          We cover all of our utilities with unit tests to make ESL the best front end component library,
+          which means every new addition works just as expected.
+      - icon: benefits/versatile.svg
+        name: Versatile
+        text: >
+          Our UI component library doesn't introduce any additional restrictions as it offers library
+          and framework-agnostic web components that work with any tech stack of your choice.
```
Comment (fshovchko) [inline pages/views/index.11tydata.yml:54]:
```suggestion
          Our UI component library doesn't introduce any additional restrictions as it offers library
          and framework-agnostic building blocks that work with any tech stack of your choice.
```

File: pages/views/index.11tydata.yml [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+      - text: Read More
+        url: /introduction/browser-support
+        style: sec-blue
+
+  promo:
+    title: Let's Get Started
+    text: >
+      Join the array of consumers who have successfully utilized our UI component library.
+      ESL brings to you a number of ultra-light reusable UI components to create unique layouts.
+      The utilities to improve development experience are right within our library too.
```
Comment (fshovchko) [inline pages/views/index.11tydata.yml:75]:
```suggestion
      Join the array of consumers who have successfully utilized our UI library.
      ESL brings to you a number of ultra-light reusable UI components to create unique layouts.
      The utilities to improve development experience are right within our library too.
```

Comment (abarmina) [inline pages/views/index.11tydata.yml:75]:
```suggestion
      Join the number of consumers who successfully use ESL.
      We bring to you a number of ultra-light reusable UI components to create unique layouts.
      The utilities to improve the development experience are also right in our library.
```

**Review by abarmina — APPROVED** [review]
it looks super!

File: pages/views/index.11tydata.yml (hunk 2 above)
Comment (ala-n) [reply pages/views/index.11tydata.yml:29]: It seems to me the current version more suits the marketing site.
Anyway, let's deliver such corrections to our content team proofreaders.

File: pages/views/index.11tydata.yml (hunk 4 above)
Comment (ala-n) [reply pages/views/index.11tydata.yml:39]: We should also cover some SEO reasons. But you're right, in general not all the keywords look natural in the text.
For that phrase let's better leave it as it is.

File: pages/views/index.11tydata.yml [hunk 8]
```diff
@@ -0,0 +1,89 @@
+seoTitle: ESL — A Web Components Library
+description: >
+  ESL is an open-source UI component library based on web components.
+  Create user-friendly interfaces and build sites from the ground up faster using framework agnostic web components.
```
[the remark this answers is not in the document]
Comment (ala-n) [reply pages/views/index.11tydata.yml:4]: Here is the specific of meta description I think.

File: pages/views/index.11tydata.yml (hunk 6 above)
[the remark this answers is not in the document]
Comment (ala-n) [reply pages/views/index.11tydata.yml:54]: Hmm..., agree with that point. Shared with the content team thanks!

File: pages/views/index.11tydata.yml (hunk 5 above)
[the remark this answers is not in the document]
Comment (ala-n) [reply pages/views/index.11tydata.yml:44]: Not completely agree with the first replacement, but the second one is under consideration to improve readability.

File: pages/views/index.11tydata.yml (hunk 7 above)
Comment (ala-n) [reply pages/views/index.11tydata.yml:75]: Some of the replacements sound reasonable to me. Shared with the content team. Thanks!

Comment (yadamskaya) [inline pages/views/index.11tydata.yml:75]:
```suggestion
      ESL brings to you a number of ultra-light reusable UI components to create unique layouts, as well as utilities to improve the development experience.
```

Comment (ala-n) [reply pages/views/index.11tydata.yml]: Thanks, applied with the latest corrections from Angela.

**Comment by ala-n** [comment]
Content submitted to proofreader's review

**Review by Anna-MariiaPetryk — APPROVED** [review]

**Review by dshovchko — APPROVED** [review]

**Pull request merged**